src/Nethermind/Nethermind.TxPool/Filters/RecoverAuthorityFilter.cs

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core;
+using Nethermind.Crypto;
+
+namespace Nethermind.TxPool.Filters
+{
+    /// <summary>
+    /// Will recover authority from transactions with authority_list and filter any with bad signatures.
```

Comment:
comment is wrong, we won't filter bad signatures here

Reply:
removed